rust_entrypoint/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+cargo-features = ["edition2021"]
+
+[package]
+name = "pyth-oracle"
```

Comment:
Could we organise the directory structure so that all of the oracle files are in the same place? Something like
- `program`
   - `c`
   - `rust`
      - `cargo.toml`
      ...

Reply:
Done